test/PowerShellEditorServices.Test/Language/LanguageServiceTests.cs

```diff
@@ -23,14 +24,17 @@ namespace Microsoft.PowerShell.EditorServices.Test.Language
 {
     public class LanguageServiceTests : IDisposable
     {
+        private readonly ITestOutputHelper output;
```

Comment:
```suggestion
        private readonly ITestOutputHelper _output;
```

Reply:
This is in a "DROP ME" commit.  please ignore.